src/containers/change-requests-table/change-requests-table.test.tsx

```diff
@@ -3,32 +3,45 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { rest } from 'msw';
-import { setupServer } from 'msw/node';
-import { render, screen } from '@testing-library/react';
-import { exampleAllChangeRequests } from '../../test-support/test-data/change-requests.stub';
+import { screen } from '@testing-library/react';
 import ChangeRequestsTable from './change-requests-table';
+import { useAllChangeRequests } from '../../services/change-requests';
+import { ApiHookReturn } from '../../services/api-request';
+import { ChangeRequest,exampleAllChangeRequests } from 'utils';
+import { routes } from '../../shared/routes';
+import { renderWithRouter } from '../../test-support/test-utils'
+import { fullNamePipe, wbsPipe } from '../../shared/pipes';
 
-const endpointURL: string = '/.netlify/functions/change-requests';
+
+jest.mock('../../services/change-requests');
 
 // Mock the server endpoint(s) that the component will hit
-const server = setupServer(
-  rest.get(endpointURL, (req, res, ctx) => {
-    return res(ctx.json(exampleAllChangeRequests));
-  })
-);
+const mockedUseAllChangeRequest = useAllChangeRequests as jest.Mock<
+  ApiHookReturn<ChangeRequest[]>
+>;
+
+const mockHook = (isLoading: boolean, errorMessage: string, responseData?: ChangeRequest[]) => {
+  mockedUseAllChangeRequest.mockReturnValue({ isLoading, errorMessage, responseData });
+};
 
 // Sets up the component under test with the desired values and renders it.
 const renderComponent: () => void = () => {
-  render(<ChangeRequestsTable />);
+  renderWithRouter(<ChangeRequestsTable />, {
+    path: routes.CHANGE_REQUESTS,
+    route: `${routes.CHANGE_REQUESTS}`
```

Comment:
You should be able to remove the template string surrounding `routes.CHANGE_REQUESTS` since the template string isn't adding anything.